### PR TITLE
feat: redesigning home page UA-238

### PR DESCRIPTION
**Pull request by ghost**
DOCS

### CODE REVIEW

**Comment by tusharkgeospoc** [comment]
Hi @vinayak,
I have 2 questions...

1.  As I'm using `@nuxt/content package` so, do I need to remove `docus package` from runtime dependencies ??

2. I'm able to redirect to `.md files` from home page scoped in `docs dir` , but those files not retaining styling as those retain in current site here - https://geospoc.github.io/unc-sch-documentation/

Can you please provide your feedback, I will make changes accordingly then, Thanks....

**Comment by tusharkgeospoc** [comment]
> Hi @vinayak,
> I have 2 questions...
> 
> 1. As I'm using `@nuxt/content package` so, do I need to remove `docus package` from runtime dependencies ??
> 2. I'm able to redirect to `.md files` from home page scoped in `docs dir` , but those files not retaining styling as those retain in current site here - https://geospoc.github.io/unc-sch-documentation/
> 
> Can you please provide your feedback, I will make changes accordingly then, Thanks....

Have removed `docus package` runtime dependency, post that `License Compliance` releated checks seems passed.
Still you can suggest if anything in this regard can be improved, Thanks...

**Comment by vinayakkulkarni** [comment]
> Hi @vinayak,
> I have 2 questions...
> 
> 1. As I'm using `@nuxt/content package` so, do I need to remove `docus package` from runtime dependencies ??
> 2. I'm able to redirect to `.md files` from home page scoped in `docs dir` , but those files not retaining styling as those retain in current site here - [geospoc.github.io/unc-sch-documentation](https://geospoc.github.io/unc-sch-documentation/)
> 
> Can you please provide your feedback, I will make changes accordingly then, Thanks....

1. Do not modify the docus settings. just add a `pages/landing.vue` and create the landing page
2. The styles etc. are applied due to docus, if you remove docus, it'll go away. simple!

File: assets/css/fonts.css [hunk 1]
```diff
@@ -0,0 +1,227 @@
+@font-face {
+  font-family: 'Inter';
+  font-style: normal;
+  font-weight: 100;
+  font-display: swap;
+  src:
+    url("~@/assets/fonts/Inter-Thin.woff2?v=3.15") format("woff2"),
+    url("~@/assets/fonts/Inter-Thin.woff?v=3.15") format("woff");
```
Comment (vinayakkulkarni) [inline assets/css/fonts.css:8]: where are the font binary files?

File: tailwind.config.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  extend: {
+    fontFamily: {
+      sans: ['Inter var', ..._fontFamily.sans],
+    },
+  },
+};
+export const variants = {
+  borderColor: ['responsive', 'hover', 'focus', 'active'],
+  ..._variants,
+};
```
Comment (vinayakkulkarni) [inline tailwind.config.js:30]: This is not needed, docus uses windicss behind the scenes, you'll have all tailwind classes out of the box

File: pages/index.vue [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    </div>
+  </div>
+</template>
+
+<script>
+  import { defineComponent } from '@nuxtjs/composition-api';
+  export default defineComponent({
+    name: 'Index',
+  });
+</script>
```
Comment (vinayakkulkarni) [inline pages/index.vue:124]: make this `landing.vue` instead of `index.vue` and let docus handle the routes for docs md file

**Review by vinayakkulkarni — CHANGES_REQUESTED** [review]

File: content/en/1.index.md [hunk 4]
```diff
@@ -1,44 +1,246 @@
 ---
 title: Introduction
+hideToc: true
+layout:
+  aside: false
+  fluid: true
+bulletLinks: ['bulletLinks', 'bulletLinks']
```
Comment (vinayakkulkarni) [inline content/en/1.index.md:7]: Can you please fix this?

**Review by vinayakkulkarni — CHANGES_REQUESTED** [review]

File: content/en/1.index.md (hunk 4 above)
Comment (tusharkgeospoc) [reply content/en/1.index.md]: @vinayakkulkarni : Made changes under this commit - https://github.com/geospoc/gqi-mapper-documentation/pull/81/commits/0a8d40df9cfe717421205cbde015ba35df09e0c0 & pushed.

Also, incorporated changes as discussion happened with @puneethshankar 
Consider review & merge of this PR, Thanks....

File: nuxt.config.ts [hunk 5]
```diff
@@ -40,6 +40,8 @@ export default withDocus({
       },
     ],
   },
+
+  modules: ['@nuxt/content'],
```
Comment (vinayakkulkarni) [inline nuxt.config.ts:44]: do you need this?

**Review by vinayakkulkarni — APPROVED** [review]
LGTM

**Pull request merged**